src/index.ts

```diff
@@ -2,12 +2,12 @@ import { createFetch } from './base'
 
 export * from './base'
 
-// ref: https://github.com/tc39/proposal-global
```

Comment:
Can we please keep this? Or do you have a more updated ref?

Reply:
Well we can mention with `this` fallback addition :p BTW we also need to update unenv https://github.com/unjs/unenv/blob/main/src/runtime/polyfill/globalThis.ts

Reply:
I can update the `unenv` if you want and use this repository as a ref comment 😄